dandiapi/api/models/asset.py

```diff
@@ -162,6 +162,13 @@ def is_blob(self):
     def is_zarr(self):
         return self.zarr is not None
 
+    @property
+    def is_embargoed(self):
```

Comment:
Pylance shows the return type here as `(bool | Any)` for me, I assume due to it not being able to infer the type of `self.zarr.embargoed`.
```suggestion
    def is_embargoed(self) -> bool:
```